database/sql/V048__create_settlement_rewards.sql

```diff
@@ -12,20 +14,20 @@ CREATE TABLE settlement_observations (
   -- table, read from the transaction receipt
   block_number bigint NOT NULL,
   log_index bigint NOT NULL,
-  gas_used numeric(78,0) NOT NULL,
-  effective_gas_price numeric(78,0) NOT NULL,
+  gas_used numeric(78,0),
+  effective_gas_price numeric(78,0),
   -- the surplus observed from the transaction call data,
   -- and converted to ETH with the auction external prices.
-  surplus numeric(78,0) NOT NULL,
-  fee numeric(78,0) NOT NULL,
+  surplus numeric(78,0),
+  fee numeric(78,0),
 
   PRIMARY KEY (block_number, log_index)
 );
 
 CREATE TABLE auction_prices (
-  auction_id bigint NOT NULL,
-  token bytea NOT NULL,
-  price numeric(78,0) NOT NULL
+  auction_id bigint NOT NULL PRIMARY KEY,
+  tokens bytea[] NOT NULL,
+  prices numeric(78,0)[] NOT NULL
 );
```

Comment:
Might not be the perfect design but seemed convenient because we always fetch all prices for an auction.

Reply:
Hmm, I think we should have a table with a single row per price. Then, we would insert multiple rows per auction. I am not a fan of using arrays like this in this table.

Reply:
Reverted.